modules/ROOT/pages/audit-logging.adoc

```diff
@@ -854,8 +854,8 @@ properties: none
 |UserID
 |n/a
 |Login
-|subaction: none
-properties: none
+|subaction: Failure
+properties: error message
```

Comment:
This problem would pre-date this PR, but I'm looking at this table in both rich text and the current output on the docs site. Should there be line breaks between the `subaction: x` and `properties: y` in those cells?

I've never changed this page, so I'm not sure how this should look...

Reply:
oh that's interesting. I was looking at it from the docs site https://docs.mulesoft.com/access-management/audit-logging and it seems like based on the column width it happens to work out most of the time but there are definitely cases where it blends together
I'm thinking it might be worth fixing all of them now?